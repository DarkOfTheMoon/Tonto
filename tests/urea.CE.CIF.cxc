
Crystal Explorer data

begin CIF             urea
file_name = /.automount/users/reaper/crystalexplorer/crystalexplorer/examples/miscellaneous/urea.cif
data_block_name =             urea
end CIF

begin crystal             urea

begin crystalcell urea
   formula = "C1 H4 N2 O1"
   spacegroup =     "p -4 21 m"
   a =        5.589000
   b =        5.589000
   c =        4.694700
   alpha =       90.000000
   beta  =       90.000000
   gamma =       90.000000
end crystalcell

begin atoms urea
               C        0.000000        5.280839        2.912578        0.000000        0.000000        0.000000        0.000000        0.000000        0.000000 IN
               O        0.000000        5.280839        5.290193        0.000000        0.000000        0.000000        0.000000        0.000000        0.000000 IN
               N        1.528275        6.809114        1.582711        0.000000        0.000000        0.000000        0.000000        0.000000        0.000000 IN
               H        2.699459        7.980299        2.527320        0.000000        0.000000        0.000000        0.000000        0.000000        0.000000 IN
               H        1.507962        6.788801       -0.323806        0.000000        0.000000        0.000000        0.000000        0.000000        0.000000 IN
end atoms
end crystal
Spartan data

=== SPARTAN DATA ===
urea
M001
0 1
        6        0.000000        2.794500        1.541270
        8        0.000000        2.794500        2.799450
        7        0.808728        3.603228        0.837534
        1        1.428492        4.222992        1.337400
        1        0.797979        3.592479       -0.171351
ENDCART
ATOMLABELS
             "C"
             "O"
             "N"
            "H1"
            "H2"
ENDATOMLABELS
