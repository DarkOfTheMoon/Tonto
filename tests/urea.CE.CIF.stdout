
---------------------------------------------------------------

 T   O   N   T   O

 A free object-based system for computational chemistry

 Version: 2.1
 Platform: INTEL-ifc-on-LINUX
 Build-date: Wed Aug 11 09:44:59 WST 2004

 For conditions of use, on-line documentation, and contributor
 and developer information see :-

 www.theochem.uwa.edu.au/tonto/

 Dylan Jayatilaka
 Daniel Grimwood
---------------------------------------------------------------

Timer started at 23/09/2004 10:56:35.

keyword found --> basis_set_directory=

keyword found --> CX_file_name=

keyword found --> CIF=

keyword found --> process_CIF_for_CX
 
Warning in routine MOLECULE.MAIN:find_CIF_crystal_data_block_1 ... Job name has now been set to: urea_urea

Cluster information:

Radius                  =        0.000000
Generation method       =        fragment
No. of atoms            =        5
No. of fragment atoms   =        5
No. of bravais shifts   =        2
Fragment width          =        0.255590        0.255590        0.632799
Cartesian width         =        2.699459        2.699459        5.613999
Fragment offset         =        0        0        0
Do fragment symop?      =        T

Fragment geometry (crystal axis system):

---------------------------------------------------------
     Atom
        #               x               y               z
---------------------------------------------------------

        1        0.000000        0.500000        0.328300
        2        0.000000        0.500000        0.596300
        3        0.144700        0.644700        0.178400
        4        0.255590        0.755590        0.284874
        5        0.142777        0.642777       -0.036499
---------------------------------------------------------

Cluster geometry (crystal axis system):

----------------------------------------------------------------------------------------------------------------------
                                                              Unit     Asym     Frag     Frag  Cluster
     Atom                                                     cell     cell     cell   parent    symop   Cluster symop
        #               x               y               z     atom     atom     atom     atom    index   s   h   k   l
----------------------------------------------------------------------------------------------------------------------

    C (1)        0.000000        0.500000        0.328300        1        1        1        1        1   1   0   0   0
    O (2)        0.000000        0.500000        0.596300        3        2        2        2        1   1   0   0   0
    N (3)        0.144700        0.644700        0.178400        5        3        3        3        1   1   0   0   0
   H1 (4)        0.255590        0.755590        0.284874        9        4        4        4        1   1   0   0   0
   H2 (5)        0.142777        0.642777       -0.036499       13        5        5        5        1   1   0   0   0
----------------------------------------------------------------------------------------------------------------------

Writing Crystal Explorer data file: /.automount/users/reaper/crystalexplorer/crystalexplorer/examples/miscellaneous/urea.cxc


Wall-clock time taken for job "urea" is 1 second, 17 milliseconds.
CPU time taken for job "urea" is 0.200 CPU seconds.
